README.md

```diff
@@ -57,6 +58,7 @@ module private_gitlab {
 | backups\_enabled | Enabled or not the automated backups | `bool` | `false` | no |
 | retention\_days | Retention in days for automated backups | `number` | `null` | no | 
 | gitlab\_snapshot\_id | Snapshot id to use for restoring an existitent Gitlab | `string` | `null` | no |
+| swap_volume_size | Size in gb of the swap volume | `string` | `""` | yes |
```

Comment:
You have to escape underscores, if not it will break the markdown syntax. Like this
`swap\_volume\_size`

Also, it says that is a string variable, but in the example you added an int

Reply:
Done, changed to number and fixed the underscores.

Reply:
Default value is wrong, it is written like an empty string. Should match with the one in variables.tf

Reply:
Done, changed to match the one in variables.tf